schemas/2018-07-01/Microsoft.Media.json

```diff
@@ -1864,6 +1867,7 @@
       "properties": {
         "firstPlayExpiration": {
           "type": "string",
+          "format": "duration",
```

Comment:
>"format": "duration", [](start = 10, length = 21)

duration format is not supported by Visual Studio... It's an issue of VS but right now we have to work around for that...

Reply:
@TianoMS Worth adding a validation to catch this?